docs/guides/walkthrough/deployment-1/index.html

```diff
@@ -919,35 +919,35 @@ <h3 id="part-1-create-the-environment">Part 1: Create the Environment<a class="h
 At the command prompt, execute the following:</p>
```

Comment:
[nit] we only update the `layer0/docs` directory on releases - it helps keep these prs smaller when we just look at `layer0/docs-src`. Doesn't need to be changed here, just for future reference.